srcpkgs/xbps/template

```diff
@@ -33,21 +33,27 @@ fi
 do_configure() {
 	./configure --prefix=/usr --sysconfdir=/etc ${CHROOT_READY:+--enable-tests}
 }
+
 post_install() {
+	local _repo="repository=https://alpha.de.repo.voidlinux.org"
 	case "$XBPS_TARGET_MACHINE" in
 	aarch64*) # XXX different repo location
-		echo "repository=https://alpha.de.repo.voidlinux.org/current/aarch64" > \
+		echo "${_repo}/current/aarch64" > \
 			${DESTDIR}/usr/share/xbps.d/00-repository-main.conf
 		;;
 	*-musl) # XXX different repo location
-		echo "repository=https://alpha.de.repo.voidlinux.org/current/musl" > \
+		echo "${_repo}/current/musl" > \
 			${DESTDIR}/usr/share/xbps.d/00-repository-main.conf
 		;;
 	*)
-		echo "repository=https://alpha.de.repo.voidlinux.org/current" > \
+		echo "${_repo}/current" > \
 			${DESTDIR}/usr/share/xbps.d/00-repository-main.conf
 		;;
 	esac
```

Comment:
Bikeshedding, I actually prefer something like:
```sh
local _postfix=
case "$XBPS_TARGET_MACHINE" in
     aarch64*) _postfix="/aarch64" ;;
     *-musl) _postfix="/musl" ;;
esac
echo "repository=https://alpha.de.repo.voidlinux.org/current$_postfix" \
    >${DESTDIR}/usr/share/xbps.d/00-repository-main.conf
```

Reply:
it's called suffix :P

Reply:
Hm, it seems like they're interchangeable https://english.stackexchange.com/questions/81263/postfix-or-suffix 
But, whatever...

Reply:
people very rarely say postfix except in specific cases (like postfix operator in programming languages), when you're appending something, it's nearly always called a suffix